libmscore/score.cpp

```diff
@@ -3430,6 +3430,31 @@ void Score::selectSimilarInRange(Element* e)
             score->select(ee, SelectType::ADD, 0);
       }
 
+//---------------------------------------------------------
+//   isStaffSpreadEnable
+//---------------------------------------------------------
+
+bool Score::isStaffSpreadEnable() const
+      {
+      // Don't use equal tests on floating numbers.
+      // (x < zeroTreshold) is considered to be 0.0.
+      const qreal zeroTreshold { 0.001 };
```

Comment:
why use an initializer list for a single qreal?

Reply:
Of the three forms of initialization (copy, direct and brace) the used (direct) brace initialization is the recommended form (according the [C++ course](https://www.learncpp.com/cpp-tutorial/variable-assignment-and-initialization/) I did some time ago). I use it more an more nowadays.
So, in fact it isn't an initializer list but a brace initialization (which also can be a list, depending on the type of variable you are initializing).